docs/source/conf.py

```diff
@@ -14,7 +14,18 @@
 
 import sys
 import os
-from pych.version import APP_NAME, APP_VERSION
+
+# copied from module/setup.py
+# This is hacky, but it means install_requires can be defined below.
```

Comment:
This comment can probably be removed, or maybe changed since install_requires is only relevant to setup.py.